cypress/e2e/content/content.spec.js

```diff
@@ -294,4 +294,22 @@ describe("Content Specs", () => {
       cy.get("[data-cy=toast]").contains("Saved a new ").should("exist");
     });
   });
+
+  describe("Media field image template", () => {
+    before(() => {
+      cy.intercept("**/search/*", { fixture: "template-media-file.json" }).as(
```

Comment:
@agalin920 how do you reckon we could go about adding a media field item with a URL instead of the usual image? Is that something we can request @markelmad or @mattezra to add on an existing content item? That was my initial idea on how to add a test to this as well but wasn't sure how that can be done aside from the templates during instance creation which isn't possible in this case.

Reply:
Updated the test to not use mocks. Also wanted to check if the image url that's on All Field Types will not be overwritten during the dev nightly sync.

Reply:
@theofficialnar data should always be overwritten in lower environments. If there is a particular state the instance should be in then that needs to be setup in production so it is the same every time we sync.

Reply:
@shrunyan already confirmed with Andres that the image url is also in prod so it should be carried over to dev on the nightly sync